lib/pbxFile.js

```diff
@@ -101,19 +100,18 @@ function detectType(filePath) {
     return filetype;
 }
 
-function defaultExtension(fileRef) {
-    var filetype = fileRef.lastKnownFileType && fileRef.lastKnownFileType != DEFAULT_FILETYPE ?
-        fileRef.lastKnownFileType : fileRef.explicitFileType;
+function defaultExtension (fileRef) {
+    var filetype = fileRef.lastKnownFileType && fileRef.lastKnownFileType != DEFAULT_FILETYPE
+        ? fileRef.lastKnownFileType : fileRef.explicitFileType;
 
-    for(var extension in FILETYPE_BY_EXTENSION) {
-        if(FILETYPE_BY_EXTENSION.hasOwnProperty(unquoted(extension)) ) {
-             if(FILETYPE_BY_EXTENSION[unquoted(extension)] === unquoted(filetype) )
-                 return extension;
+    for (var extension in FILETYPE_BY_EXTENSION) {
+        if (FILETYPE_BY_EXTENSION.hasOwnProperty(unquoted(extension))) {
+            if (FILETYPE_BY_EXTENSION[unquoted(extension)] === unquoted(filetype)) { return extension; }
```

Comment:
Ugh, why does the auto-fix format that onto one line :unamused: There are a few instances of changes like this. No deal breakers but not pretty either.